charts/cargohold/templates/configmap.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: "cargohold"
 data:
   cargohold.yaml: |
+    logNetStrings: True # log using netstrings encoding: http://cr.yp.to/proto/netstrings.txt
```

Comment:
Is this potentially something we'd ever want to change? If yes; we should add it to values.yaml; if no, then why is it configurable? Haha

Reply:
> if no, then why is it configurable

We don't want netstrings logging in the demo, but we want it in production, and k8s is for production only. (Actually, is there a good reason to not run the demo with minikube or k3s locally?)

Reply:
Sounds good; I was just curious :) If you have enough memory on your machine I don't see why not; but that depends on how scaled down the demo is. Local k8s can be a pretty big resource drain.